lib/govuk/registers/country.rb

```diff
@@ -12,7 +12,7 @@ def self.countries
             body = RestClient.get(URL, params: params)
             countries_page = JSON.parse(body)
 
-            results.concat(countries_page)
+            results.concat(countries_page.to_a)
```

Comment:
The `#to_a` call introduces a transformative step into what `Govuk::Registers::Country` returns. The service gives us a hash of country codes to country details, and we're transforming this to an array of `[country_code, country_details]`. This makes it easy from the point of view of using `.map` in `DfidTransition::Patch::Countries#transform_to_label_value` (which we then go on to do), but we should consider that we're altering a notionally pure service to suit our own ends, which (although trivial in this case) does amount to coupling the service results to our one use case.

If we wanted to avoid this (and return the hash that the countries register now seems to intend), we might elect to `#merge` instead, then call `#each_pair` from [DfidTransition::Patch::Countries#transform_to_label_value](https://github.com/alphagov/dfid-transition/pull/9/commits/ab9752b2b6fd7aa2c3833947972c30f56a1d4987#diff-fdc19056ad0794002d084845cfbf440cR16)